internal/mage/engine.go

```diff
@@ -310,15 +310,24 @@ func (t Engine) Dev(ctx context.Context) error {
 	runArgs := []string{
 		"run",
 		"-d",
-		// "--rm",
+	}
+
+	// Make all GPUs visible to the engine container if the GPU support flag is set:
+	if gpuSupportEnabled := os.Getenv(util.GPUSupportEnvName); gpuSupportEnabled != "" {
+		runArgs = append(runArgs, []string{"--gpus", "all"}...)
+	}
+	runArgs = append(runArgs, []string{
+		"--rm",
```

Comment:
nit: actually just delete `--rm` I think, not sure why it was commented out, but it's useful to not remove the dev engine if it dies because you can still look at the logs if it crashed.

Reply:
Fixed a few weeks ago